packages/utils/ast-utils.ts

```diff
@@ -626,14 +637,27 @@ function parseMerge(j: JSCodeshift, ast: Node, value: string, action: string): b
 				type: "MemberExpression"
 			},
 			operator: "=",
-			right: j.callExpression(j.identifier("merge"), [j.identifier(value), exportsDecl.pop()]),
+			right: j.callExpression(j.identifier("merge"), [j.identifier(configIdentifier), exportsDecl.pop()]),
 			type: "AssignmentExpression"
 		};
+		
 		(p.value as Node).body[bodyLength - 1] = newVal;
 		return false; // TODO: debug later
 	}
+
+	function addMergeImports(configIdentifier: string, configPath: string) {
+		ast.find(j.Program).forEach(p => {
+			if (!isImportPresent(j, ast, 'webpack-merge')) {
+				(p.value as Node).body.splice(-1, 0, `const merge = require('webpack-merge')`);
+			}
+			(p.value as Node).body.splice(-1, 0, `const ${configIdentifier} = require('${configPath}')`);
```

Comment:
you need to check if this import is present too, look for the configIdentifier in the body. So a new `if` here: `if importPresent(configIdentifier)`